patronload/s3.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def delete_objects_from_bucket_with_prefix(  # type: ignore
```

Comment:
Per the [boto3-stubs docs](https://youtype.github.io/boto3_stubs_docs/mypy_boto3_s3/client/), the correct way to type hint this is
```
from mypy_boto3_s3.client import S3Client

...

s3_client: S3Client
```

Reply:
Just solved this, I had the dependency in `[dev-packages]`

Reply:
Right, got it. That makes sense.